lib/api/collection_config.rb

```diff
@@ -0,0 +1,56 @@
+module Api
+  class CollectionConfig
+    def initialize
+      @cfg = Settings.collections
+    end
+
+    def [](collection_name)
+      return nil unless @cfg[collection_name.to_sym]
+      Collection.new(@cfg[collection_name.to_sym])
+    end
+
+    def names_for_feature(product_feature_name)
+      names_for_features[product_feature_name]
+    end
+
+    def name_for_klass(resource_klass)
+      @cfg.detect do |_, spec|
+        spec[:klass] && spec[:klass].constantize == resource_klass
+      end.try(:first)
+    end
```

Comment:
we should have tests around the methods to be used outside the API (i.e. central admin to use this).